src/index.ts

```diff
@@ -209,6 +219,60 @@ export function buildStorageKeySuffix(apiKey: string): string {
   return apiKey.replace(/\W/g, '').substring(0, 8);
 }
 
+/**
+ * Initializes the Eppo client with configuration parameters.
+ *
+ * The purpose is for use-cases where the configuration is available from an external process
+ * that can bootstrap the SDK.
+ *
+ * This method should be called once on application startup.
+ *
+ * @param config - client configuration
+ * @returns a singleton client instance
+ * @public
+ */
+export function initSync(config: IClientConfigSync): IEppoClient {
+  try {
+    const memoryOnlyConfigurationStore = configurationStorageFactory({
+      forceMemoryOnly: true,
+    });
+    memoryOnlyConfigurationStore.setEntries(config.flagsConfiguration);
+    EppoJSClient.instance.setConfigurationStore(memoryOnlyConfigurationStore);
+
+    // Allow the caller to override the default obfuscated mode, which is false
+    // since the purpose of this method is to bootstrap the SDK from an external source,
+    // which is likely a server that has not-obfuscated flag values.
+    EppoJSClient.instance.setIsObfuscated(config.isObfuscated ?? false);
+
+    if (config.assignmentLogger) {
+      EppoJSClient.instance.setLogger(config.assignmentLogger);
+    }
+
+    // There is no SDK key in the offline context.
+    const storageKeySuffix = buildStorageKeySuffix('no_api_key');
```

Comment:
Got another idea for handling this?

Reply:
just use a random string instead, I think that would be cleaner

Reply:
The only thing I don't love about this:

* generate a random string in JS seems like so much code `[...Array(8)].map(() => Math.random().toString(36)[2]).join('');` but that's fine
* if we move to having a persistent store i don't want to generate a different random string each initialization which would make the store useless. perhaps a different static string like `offline` is more clear?

Reply:
yeah I think either should be fine, as long as the intention is clear. could be as simple as extracting to a function like `generateStorageCacheKeySuffixForOffline` or something like that